### DISCUSSION TITLE
Difference between TM reading and car reading (energy consumption)

### DISCUSSION THREAD

**Discussion by stas87** [teslamate-org/teslamate#1506] [Q&A]
Hi,

Just returned from a 87.7km trip where car shows avg consumption of 136Wh/km, when checking same trip in TM I see net consumption of 150 wh/km (gross 153 wh/km).

Why is the difference? Which one more accurate?

And another question, what are the diffs between gross and net consumption?

Thanks

**Comment by petariga** [comment]
The value in a car is always heavily underestimated. The Tesla counts only direct consumption during driving. However, there is much more to be counted in (consumption during pauses, preheating, precooling, sentry, losses during charges a.s.o.). The difference between the display in a car and reality measured on the wattmeter is usually 20-25%.

**Reply by stas87** [reply]
Thanks!

And what is the diff between net and gross consumption?

**Reply by SteveDonie** [reply]
I was also wondering what the difference is in these two values.

**Reply by NeskireDK** [reply]
I'm just guessing here, but generally speaking.
Gross will include transmission loss, loss to heat / cold, etc. (including resistance losses in battery and loss to battery heat perhaps?) 
Net will be actually used energy for the appliance (i.e. The car, including electrics, engine, heater, and so on).
